### DISCUSSION TITLE
Monai models not available ...

### DISCUSSION THREAD

**Discussion by dcsst29** [Project-MONAI/monai-deploy-app-sdk#412] [Q&A]
Was trying to run the liver_segmentation operator, through monai-app-sdk. 
The model is not available online. 

https://ngc.nvidia.com/catalog/models/nvidia:med:clara_pt_liver_and_tumor_ct_segmentation

Can someone help... 

Thanks

**Comment by MMelQin** [comment]
Thank you @dcsst29 for the question.

It does appear that the link is not valid anymore.

The model is embedded in the MONAI Application Package (MAP), published on GitHub Container Registry, ghcr.io/mmelqin/monai_ai_livertumor_seg_app:1.0. So,
- Please pull this container, and copy out the model in the container's internal folder at `/opt/monai/models` by running the container image with entrypoint set to` /bin/bash` and volumes mapped  
- or please see this[ discussion topic](https://github.com/Project-MONAI/monai-deploy-app-sdk/discussions/340) which contains a section on copying a folder from a container without needing to run it.
